pyBumpHunter/BumpHunter.py

```diff
@@ -286,415 +336,433 @@ def Reset(self):
-        
+
         This method share most of its parameters with the BumpScan method.
-        
+
         Arguments :
             sig : Numpy array containing the simulated signal. This distribution will be used to perform the signal
                   injection.
-            
+
             bkg : Numpy array containing the expected background. This distribution will be used to build the data in
                   which signal will be injected.
-            
+
             is_hist : Boolean that specify if the given signal and background are already in histogram form. If true,
                       data and bkg are histograms and the Bins argument is expected to be a array with the bound of
                       the bins and the Weights argument is expected to be a scalar scale factor or None.
                       Default to False.
-            
-        Result inner variables :        
+
+        Result inner variables :
             signal_ratio : Ratio signal_min/signal_exp (signal strength). If signal_exp is not specified, default to None.
-            
+
             data_inject : Data obtained after injecting signal events in the backgound.
-            
+
             sigma_ar : Numpy array containing the significance values obtained at each step.
-        
+
         All the result inner variables of the BumpHunter instance will be filled with the results of the scan permormed
         during the last iteration (when sigma_limit is reached).
-        '''
-        
+        """
+
         # Set the seed if required (or reset it if None)
         np.random.seed(self.seed)
-            
+
         # Internal variables
         i = 1
         strength = 0
         data = []
-        
+
         # Reset significance and sigma_ar global variable
         self.significance = 0
         sigma_inf = 0
         sigma_sup = 0
         self.sigma_ar = []
-        
+
         # Check the expected number of signal event
-        if(self.signal_exp is None):
-            if(is_hist is False):
+        if self.signal_exp is None:
+            if is_hist is False:
                 self.signal_exp = sig.size
             else:
                 self.signal_exp = sig.sum()
-        
+
         # Turn the background distributions into histogram
-        if(is_hist is False):
-            bkg_hist,bins = np.histogram(bkg,bins=self.bins,range=self.rang,weights=self.weights)
+        if is_hist is False:
+            bkg_hist, bins = np.histogram(
+                bkg, bins=self.bins, range=self.rang, weights=self.weights
+            )
         else:
             if self.weights is None:
                 bkg_hist = bkg
             else:
                 bkg_hist = bkg * self.weights
             bins = self.bins
-        
+
         # Generate pseudo-data by sampling background
-        print('Generating background only histograms')
+        print("Generating background only histograms")
         Nbkg = 1000
```

Comment:
Is this 1000 a fixed number? Unfamiliar with the paper

Reply:
Actually, this hardcoded `Nbkg = 1000` is also coming from the c++/ROOT version I used as reference point.
It was hardcoded the same way in there, so I kept it in order to compare results.

Reply:
I see, I guess one needs to compare with the paper in order to justify this choice